src/main/scala/cromwell/CommandLineParser.scala

```diff
@@ -0,0 +1,106 @@
+package cromwell
+
+import com.typesafe.config.ConfigFactory
+import cromwell.core.path.{DefaultPathBuilder, Path}
+import scopt.OptionParser
+
+object CommandLineParser extends App {
+
+  sealed trait Command
+  case object Run extends Command
+  case object Server extends Command
+
+  case class CommandLineArguments(command: Option[Command] = None,
+                                  workflowSource: Option[Path] = None,
+                                  workflowInputs: Option[Path] = None,
+                                  workflowOptions: Option[Path] = None,
+                                  workflowType: Option[String] = Option("WDL"),
+                                  workflowTypeVersion: Option[String] = Option("v2.0-draft"),
+                                  labels: Option[Path] = None,
+                                  imports: Option[Path] = None,
+                                  metadataOutput: Option[Path] = None
+                                 )
+
+  lazy val cromwellVersion = ConfigFactory.load("cromwell-version.conf").getConfig("version").getString("cromwell")
+
+  case class ParserAndCommand(parser: OptionParser[CommandLineArguments], command: Option[Command])
+
+//   cromwell 28-5071e39-SNAP
+//
+//   Usage: cromwell [server|run] [options] <args>...
+//
+//     --help                   Cromwell - Lord Protector / Workflow Execution Engine
+//     --version
+//   Command: server
+//   Starts a web server on port 8000.  See the web server documentation for more details about the API endpoints.
+//   Command: run [options] workflow-source
+//   Run the workflow locally and print out the outputs in JSON format.
```

Comment:
yeah I copied that over from the previous version of the code which was probably pre-JES backend 😦